### DISCUSSION TITLE
Editor is missing Hex search mode

### DISCUSSION THREAD

**Discussion by lprot** [doublecmd/doublecmd#1751] [Suggestions]
### Version

Double Commander
Version: 1.2.0 alpha
Revision: 11393
Commit: e650495a5
Build date: 2024/07/06
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 12.7.5

### Description

DC is considered to be better than Total Commander but has inconsistancy in the functionality.
In viewer we have hex search option: 
![image](https://github.com/user-attachments/assets/5f842d3f-ff49-4456-9dfc-f746477f54d6)

and in the editor not:
![image](https://github.com/user-attachments/assets/77b0f29b-d35e-4542-898e-6982759378cf)

of course an advanced user can enable regex and multiline and search like \x22FF or \x22\xFF but this is not intuitive as leads to confusion https://github.com/doublecmd/doublecmd/issues/1748

Maybe regular hex search option makes more sense?

**Comment by alexx2000** [comment]
This is impossible. Viewer has a hex-mode, editor not. Viewer opens file as is, editor converts encoding to UTF-8.
